guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,389 @@
+- with a certain client
+- on Nostr and
+- on her website
+
+When she receives payments from any of these sources, the respective label(s) will be detected in the on-chain address and used to infer the source of the payment.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work-mobile@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work@2x.png"
+   alt-text = "An image that illustrates how labels feature in silent payments works."
+   caption = "Alice customizes her static address with labels before she shares them. Whenever she receives a payment from one of them, the label is auto-applied and detected by the wallet."
+   width = 1024
+   height = 801
+   layout = "full-width"
+%}
+
+This scheme improves labelling in general since labels get auto-applied to all derived on-chain addresses. This eliminates the friction of manually adding labels to addresses or transactions. Labels can contain crucial information that is useful for automatic or manual coin selection for future outgoing payments.
+
+#### Tracking payments
+
+Without using labels, receivers have no way of knowing who may have paid them since they do not interact with the sender to provide them a on-chain address.
+
+{% include tip/recommendation.html %}
+
+Labels in silent payments are critical if receivers such as businesses need to match payments with customers.
+
+{% include tip/close.html %}
+
+Here's how businesses such as exchanges, merchants and vendors can use labels:
+- Invoices: merchant software like [BTCPay](https://btcpayserver.org/) must use labels to match incoming payments with invoices and/or provide static addresses to repeat customers.
+- Bitcoin exchange deposits: Customers looking to sell/deposit bitcoin at exchanges encounter friction when their deposit addresses change constantly. Labelled static addresses for users fix this without address reuse while retaining the ability to match deposits to customers.
+
+
+### Contacts
+
+ Since senders can safely use the same static address for multiple payments, it is natural for them to want to store these for future use. Contacts are a great way to do this in terms that users can intuit: names and faces. The [contacts](/guide/daily-spending-wallet/contacts/) page provides guidance about the topic.
+
+ Silent payments (along with bolt-12) allow applications to center their payments experience around people instead of addresses, something that was not advisable before due to issues with on-chain address reuse.
```

Comment:
```suggestion
 Silent payments (along with [BOLT-12](http://bolt12.org)) allow applications to center their payments experience around people instead of addresses. This was not advisable before, due to issues with on-chain address reuse.
```